src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,79 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.InvalidDataException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    public User getDefaultValidUser() {
+        User user = new User();
+        user.setLogin("validLogin");
+        user.setId(38628L);
+        user.setPassword("unuwjf8j2");
+        user.setAge(22);
+        return user;
+    }
+
+    @Test
+    void registrationDataIsValid() throws InvalidDataException {
+        RegistrationService registrationService = new RegistrationServiceImpl();
```

Comment:
do not create a new service each time make it a class-level variable, we have been doing it this way for several weeks now